### PR TITLE
Feature/lesson 5 npm

### PR DESCRIPTION
**Pull request by anton-kan**
Added ESLint and fixed its warnings; Implemented math actions output in DOM using Browserify

### CODE REVIEW

File: js/math.js [hunk 1]
```diff
@@ -11,8 +11,6 @@ const actionSelectElement = document.querySelector('#action-select');
 const calculateBtnElement = document.querySelector('#calculate-btn');
 const resultInputElement = document.querySelector('#result-input');
 
-calculateBtnElement.addEventListener('click', calculate);
-
 function calculate() {
```
Comment (bjatta) [inline js/math.js]: const calculate = () => {

File: package.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "build": "browserify ./js/math.js -o ./js/bundle.js",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint ./js",
+    "lint:fix": "eslint ./js --fix"
```
Comment (bjatta) [inline package.json]: is it added to IDE?

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
```
Comment (bjatta) [inline .gitignore:1]: add node_modules to .gitignore

File: js/math.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+const aInputElement = document.querySelector('#a-input');
+const bInputElement = document.querySelector('#b-input');
+const cInputElement = document.querySelector('#c-input');
+const actionSelectElement = document.querySelector('#action-select');
+const calculateBtnElement = document.querySelector('#calculate-btn');
+const resultInputElement = document.querySelector('#result-input');
+
+function calculate() {
+  const a = +aInputElement.value;
```
Comment (bjatta) [inline js/math.js]: aInputElement.?value

File: package.json (hunk 2 above)
Comment (anton-kan) [reply package.json]: Yep.
image

File: .gitignore (hunk 3 above)
Comment (anton-kan) [reply .gitignore:1]: It is there. I had some inconsistency with end-of-line characters between lint and git settings, that's why git thought I removed it and then added it back, while in fact I didn't touch it.

File: js/math.js (hunk 1 above)
Comment (anton-kan) [reply js/math.js]: fixed

File: js/math.js (hunk 4 above)
Comment (anton-kan) [reply js/math.js]: fixed

**Review by bjatta — APPROVED** [review]

**Pull request merged**